src/blog/adaptor-frame.md

```diff
@@ -0,0 +1,315 @@
+---
+title: 'Faster JavaScript calls'
+author: 'Victor Gomes, the frame shredder'
```

Comment:
Do you have a twitter you want to link here?

Reply:
Nope. :)